.github/trigger.sh

```diff
@@ -32,7 +32,7 @@ case $1 in
         fi
         wip=false
         branch=$2
-        ;&
+        ;;
```

Comment:
This breaks the script. You can no longer deploy releases. You are hit by <https://en.wikipedia.org/wiki/GNU_General_Public_License#Legal_barrier_to_app_stores>.  Apple won't update bash to version 4.0. You could use the zsh instead.

Reply:
Done